device-types/Nokia/7210-SAS-M-24F-2XFP.yaml

```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-M 24F 2XFP
+slug: nokia-7210-sas-m-24f-2xfp
+part_number: 3HE05828AAAB01
+u_height: 2
+is_full_depth: false
+subdevice_role: parent
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 450
+  - name: PS2
+    type: iec-60320-c14
+    maximum_draw: 450
```

Comment:
Is the power on these modular? Or are the power ports not removable?

Reply:
Chose to remove the "power-ports" section and add the module-bays only